README.md

```diff
@@ -1,30 +1,43 @@
 # Cleanapp Backend version 2+
 
 This repository is for CleanApp (http://cleanapp.io) backend development.
-It's a complete rewrite after v.0.
 
-## Installation
+# Installation
 
-Pre-requisites: Linux (Debian/Ubuntu/...), this is tested on Google Drive Ubuntu VPS instance.
+## Pre-requisites
+
+1.  Make sure that your local machine has Docker installed. https://docs.docker.com/engine/install/
+1.  Make sure you're prepared for working with GoogleCloud.
+    1.  You got necessary access to Google Cloud services. Ask project admins for them.
+    1.  You have gcloud command line interface installed, https://cloud.google.com/sdk/docs/install
+    1.  You are successfully logged in gcloud, https://cloud.google.com/sdk/gcloud/reference/auth/login
+
+## Build Docker image
+
+1.  Modify the Docker image version if necessary. Open the file `docker/.version` and set the desired value of the `BUILD_VERSION`.
+1.  Run the `docker/build_server_image.sh` from the `docker` directory.
+    ```
+    cd docker &&
+    ./build_server_image.sh
+    ```
+
+## Deploying in Google Cloud
+
+Pre-requisites: Linux (Debian/Ubuntu/...), this is tested on Google Cloud Ubuntu VPS instance.
 
 1. Login to the target machine.
    * On GCloud you go to the dashboard, pick the instance, and the click on SSH
 1. Get setup.sh into the current directory, e.g. using
 ```shell
-curl https://raw.githubusercontent.com/cleanappio/cleanapp_back_end_v2/main/setup/setup.sh > setup.sh
+curl https://raw.githubusercontent.com/cleanappio/cleanapp_back_end_v2/main/setup/setup.sh > setup.sh &&
+sudo chmod a+x setup.h
 ```
-3. In case you use non-standard Docker images, open setup.sh and edit two Docker params at the top.
-4. Run
+1. Run
 ```
 ./setup.sh
 ```
-5. When doing it, you will be asked to set the DB passwords:
```

Comment:
Now it's good to remove it, but please, explain from where passwords are taken and a link how to manage them.

Reply:
Same here. The README wasn't changed in this PR.

What actually happened is that there were two PRs in two branches, this one was branched incl. changes in the previous branch. After merging the previous branch and merging this one with main after that this branch doesn't contain README changes anymore.